src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs

```diff
@@ -105,8 +109,7 @@ public RestoreCommand(IApplicationEnvironment env)
                         Interlocked.Increment(ref successCount);
                     }
                 };
-
-                if (PlatformHelper.IsMono)
+                if (!RestoreInParallel || PlatformHelper.IsMono)
```

Comment:
is `PlatformHelper.IsMono` special case still needed?  `RestoreInParallel` defaults to `false`